packages/.vscode/launch.json

```diff
@@ -22,7 +22,9 @@
             "type": "dart",
             "program": "devtools_app/lib/main.dart",
             "args": [
-                "--dart-define=enable_experiments=true"
+                "--dart-define",
+                "memory_offline_experiment=true",
```

Comment:
does this work? we might need a comma after '=true'

Reply:
It worked. But it seems there is better way: https://dartcode.org/docs/using-dart-define-in-flutter/